### DISCUSSION TITLE
Osu not opening until 3 or 4 times

### DISCUSSION THREAD

**Discussion by PPie236** [ppy/osu#22764] [Q&A]
I made a discussion on the osu forums to no avail, I was told to post my log files here 

[database.log](https://github.com/ppy/osu/files/10867733/database.log)
[input.log](https://github.com/ppy/osu/files/10867735/input.log)
[network.log](https://github.com/ppy/osu/files/10867736/network.log)
[performance.log](https://github.com/ppy/osu/files/10867738/performance.log)
[runtime.log](https://github.com/ppy/osu/files/10867739/runtime.log)
[updater.log](https://github.com/ppy/osu/files/10867740/updater.log)

**Comment by Axilotl17** [comment]
related to #21658 maybe? although there is no data in the logs for some reason...

**Reply by PPie236** [reply]
it seems to be different than that I don't know why there's no data though there seems to be data in the files themselves although I don't know what any of it means ill try to repost it otherwise will a screenshot work?

[database[1].log](https://github.com/ppy/osu/files/10873854/database.1.log)
[input[1].log](https://github.com/ppy/osu/files/10873855/input.1.log)
[network[1].log](https://github.com/ppy/osu/files/10873858/network.1.log)
[performance[1].log](https://github.com/ppy/osu/files/10873859/performance.1.log)
[runtime[1].log](https://github.com/ppy/osu/files/10873861/runtime.1.log)
[updater[1].log](https://github.com/ppy/osu/files/10873863/updater.1.log)

**Reply by Axilotl17** [reply]
the initial logs you sent were empty. these are not.
however these appear to be different logs. can you try deleting all the logs in the folder, and open the game, and if it fails to launch send the newly created log files.

**Reply by PPie236** [reply]
it seems to be fixed all of the sudden will updates you just in case
